dataland-keycloak/dataland_theme/email/html/email-verification.ftl

```diff
@@ -1,12 +1,12 @@
 
```

Comment:
The feedback in the frontend where you get to know that you received a verification mail should say "preview account" too:

![image](https://user-images.githubusercontent.com/99255296/197747732-8a83023d-eeec-4461-adae-da831df359c4.png)

Reply:
If you can't find this text segment in the code, then the reason is that it is the default keycloak text.
It should be possible to change it by customizing the text value of the respective message in the theme properties.

Reply:
hopefully fixed